google/cloud/speech/cloud_speech.yaml

```diff
@@ -12,18 +12,8 @@ documentation:
 
 apis:
 - name: google.cloud.speech.v1beta1.Speech
-# TODO get LRO API call support added for Speech
-#- name: google.longrunning.Operations
+- name: google.longrunning.Operations
```

Comment:
Yes that is possible, but it's not really the intended design of Operations - it's meant to be the same interface when used in multiple APIs, in contrast with the way IAM works.